PlaylistManager/HarmonyPatches/MainSystemInit.cs

```diff
@@ -0,0 +1,15 @@
+﻿using HarmonyLib;
+using PlaylistManager.Utilities;
+using Zenject;
+
+namespace PlaylistManager.HarmonyPatches
+{
+    internal class MainSystemInitPatch
+    {
+        [HarmonyPatch(typeof(MainSystemInit), nameof(MainSystemInit.InstallBindings))]
+        private static void Postfix(DiContainer container)
+        {
+            PlaylistLibUtils.coroutineStarter = container.Resolve<ICoroutineStarter>();
```

Comment:
Might replace the use of `CoroutineStarter` by `UnityMainThreadTaskScheduler`? I am not sure they're equivalent.